lib/VerifiableCredentialStore.js

```diff
@@ -128,7 +129,7 @@ export default class VerifiableCredentialStore {
       if(!doc) {
         return false;
       }
-      return this.edv.delete({id: doc.id, invocationSigner});
+      return this.edv.delete({doc, invocationSigner});
```

Comment:
good catch, here's the edv-client delete API for inquiring minds: https://github.com/digitalbazaar/edv-client/blob/master/EdvClient.js#L370-L372